.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: Build and Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-version:
+        description: 'Version (and git tag) that this release should use'
```

Comment:
Can we please have an `e.g. 1.2.3` example on what is expected here? `23.7.0`?

Can we please also clean up the tags in https://github.com/stackabletech/druid-opa-authorizer/tags (e.g. after merging this PR)?